en/docs/how-tos/passing-enduser-attributes-to-the-backend-using-jwt.md

```diff
@@ -175,6 +175,69 @@ Outer `[jwtGeneratorConfig]` configuration contains the properties and the custo
 </tbody>
 </table>
 
+#### Adding Claims Retrieved from a remote endpoint
```

Comment:
Can you fix the above heading "Extract backed token from the jwt access token" to "Extract **backend** token from the **JWT** access token"
Also there are few typos in the existing doc. please fix them too. 
1. dependency -> dependency
2. developers have the capability use -> developers have the capability to use 
3. jwt -> JWT
4. Since, -> Since
5. if the authentication token consist the backendJwt claim in the payload, value of the backendJwt -> if the authentication token consists of the backendJwt claim in the payload, the value of the backendJwt

Reply:
updated via https://github.com/wso2/docs-mg/pull/43/commits/9480fc53f208b95446e6f3e220351d4b5229fa1c